server/plugin.go

```diff
@@ -49,21 +51,36 @@ func (p *Plugin) OnActivate() error {
 	teamSplit := split[0]
 	channelSplit := split[1]
 
-	team, err := p.API.GetTeamByName(teamSplit)
-	if err != nil {
-		return err
+	team, getTeamError := p.API.GetTeamByName(teamSplit)
+	if getTeamError != nil {
+		return getTeamError
 	}
 	p.TeamID = team.Id
 
-	user, err := p.API.GetUserByUsername(p.configuration.Username)
+	botID, ensureBotError := p.Helpers.EnsureBot(&model.Bot{
```

Comment:
when the error is of type `error`, the go idiomatic convention is to use variable name `err`.  Let's use that here, instead of `ensureBotError`